rice-namenode/source/ClientNamenodeProtocolImpl.cc

```diff
@@ -110,8 +110,17 @@ std::string ClientNamenodeTranslator::getFileInfo(std::string input) {
   req.ParseFromString(input);
   logMessage(&req, "GetFileInfo ");
   GetFileInfoResponseProto res;
-  zk->get_info(req, res);
-  logMessage(&res, "GetFileInfo response ");
+  switch (zk->get_info(req, res)) {
+    case zkclient::ZkNnClient::GetFileInfoResponse::Ok:
+      logMessage(&res, "GetFileInfo response ");
```

Comment:
Should this use the `LOG` macro for consistency?

Reply:
Thanks for the review! I think the intent was to log the contents of of a successful proto response, although logMessage isn't actually doing that at this point since the response pointer is unused in logMessage. I suppose making logMessage actually log the contents of the response could go into a separate pull request.